pkg/endpoint/api.go

```
@@ -240,14 +240,15 @@ func (e *Endpoint) GetModelRLocked() *models.Endpoint {
 		spec.Options = *e.Options.GetMutableModel()
 	}
 
+	securityIdentity, _ := e.GetSecurityIdentity()
```

I find it suspicious to ignore the error here.